### PR TITLE
Move "Ergonomic brand checks for private fields" to ES2022

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
They reached Stage 4 in the last meeting

### CODE REVIEW

**Review by ljharb — APPROVED** [review]

File: data-es2016plus.js [hunk 1]
```diff
@@ -5714,6 +5714,36 @@ exports.tests = [
...
+    name: 'Ergonomic brand checks for private fields',
+    category: '2022 features',
+    significance: 'small',
+    spec: 'https://github.com/tc39/proposal-private-fields-in-in',
+    exec: function () {/*
+      class A {
+        #x;
+        static check(obj) {
+          return #x in obj;
+      rhino1_7_13: false
```
Comment (nicolo-ribaudo) [inline data-es2016plus.js]: Whops, this is wrong (it was also there before this PR)

```suggestion
```

Comment (ljharb) [reply data-es2016plus.js]: nice catch; it'd have been a noop at least, so it wouldn't break anything.

Comment (nicolo-ribaudo) [reply data-es2016plus.js]: Oh right, I didn't even realize it's a valid labeled statement.

**Review by chicoxyzzy — APPROVED** [review]
This needs rebase now. Sorry for that. Otherwise it looks good

**Comment by nicolo-ribaudo** [comment]
Rebased!

**Pull request merged**